securedrop/server_os.py

```diff
@@ -12,3 +13,25 @@ def get_os_release() -> str:
         stdout=subprocess.PIPE,
         universal_newlines=True
     ).stdout.strip()
+
+
+@functools.lru_cache()
+def get_tor_version() -> str:
+    try:
+        output = subprocess.run(
+            ["/usr/bin/tor", "--version"],
```

Comment:
Is this gonna require an apparmor ruleset change? (dev doesn't have it so staging test required)